content/en-us/reference/engine/enums/RollOffMode.yaml

```diff
@@ -16,20 +16,23 @@ items:
     summary: |
       Volume attenuates from `Class.Sound.RollOffMinDistance` in an inverse
       manner. This option mirrors how sounds attenuate in the real world.
+      This is done through `Class.Sound.RollOffMinDistance`/`Distance`
```

Comment:
What is `Distance` here (and in following suggestions) referring to specifically?

Reply:
Distance refers to the (Vector3 Position of the Audio Source - Vector3 Position of the Audio Listener).Magnitude